.github/actions/bundle/src/helpers.ts

```diff
@@ -1,5 +1,5 @@
 export const getImport = (line: string): { importedFile: string; isImport: boolean } => {
-    const matches = line.match(/require\(["']([A-Z_a-z.]+)["']\)/iu)
+    const matches = line.match(/require[\( ]?["']([A-Z_a-z.]+)["']/iu)
```

Comment:
Yes, I was aware of that but I figured that it wasn't the role of the bundler to check for Lua syntax errors, since the existing code ignored things like incorrectly matched opening/closing quotes.

Anyway, I updated the code with a more complete solution which handles every case I can think of except:
- Comments (eg `--local foo = require("bar")`. There would be a lot of unnecessary duplication between this and the `remove-comments` module if that were handled here and I think it would make more sense to strip the comments before parsing imports.
- `luasocket`. I'd have to look into it.

Let me know if that's okay. I still need to install `node`